lib/class-wp-duotone-gutenberg.php

```diff
@@ -444,4 +859,16 @@ public static function migrate_experimental_duotone_support_flag( $settings, $me
 
 		return $settings;
 	}
+
+	/**
+	 * Gets the SVG for the duotone filter definition from a preset.
+	 *
+	 * @param array $preset The duotone preset.
+	 * @return string The SVG for the filter definition.
+	 */
+	public static function get_filter_svg_from_preset( $preset ) {
+		// TODO: This function will be refactored out in a follow-up PR where it will be deprecated.
```

Comment:
Deprecated in https://github.com/WordPress/gutenberg/pull/49702/files#diff-c6cf688aa7b50316afa7fc57327b6ac81c1ac831226447c53ffc88c32fc23750L870-L871